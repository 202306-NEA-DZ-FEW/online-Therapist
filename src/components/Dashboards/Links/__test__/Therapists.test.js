import renderer from "react-test-renderer";

import TherapistsMatches from "../Therapists";

it("renders correctly", () => {
    const tree = renderer.create(<TherapistsMatches url='' />).toJSON();
    expect(tree).toMatchSnapshot();
});
