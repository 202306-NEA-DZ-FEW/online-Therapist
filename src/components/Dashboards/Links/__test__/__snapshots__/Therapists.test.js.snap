// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="font-atkinson p-2"
>
  <h1
    className="text-2xl md:text-4xl font-bold bg-Teal p-4 md:p-6  w-full text-white uppercase rounded-lg tracking-wider "
  >
    therapists.heading
  </h1>
  <ul
    className="flex flex-col gap-y-8 lg:m-4"
  />
</div>
`;
